File: aspnet-core/src/ERP.Application.Shared/PayRoll/SalaryLock/Dtos/SalaryLockDto.cs
using System;
using Abp.Application.Services.Dto;

namespace ERP.PayRoll.SalaryLock.Dtos
{
    public class SalaryLockDto : EntityDto
    {
        public int? TenantID { get; set; }

        public int? SalaryMonth { get; set; }

        public int? SalaryYear { get; set; }

        public bool Locked { get; set; }

        public DateTime? LockDate { get; set; }

    }
}

File: angular/src/app/main/payroll/shared/dto/salarylock-dto.ts
export class salarylockDTo{
    id:number;
    tenantID?: number;
    salaryMonth?: number;
    salaryYear?: number;
    locked: boolean;
    jvLocked: boolean;
    lockDate?: string;
}




export class GetsalarylockForViewDto {
    salarylock: salarylockDTo
}

export class GetsalarylockEditOutput {
    salaryLock: salarylockDTo
}

export class PagedResultDtosalarylock {
    totalCount: number;
    items: GetsalarylockForViewDto[]
}
export class CreateOrEditsalarylockDto {
    
  id:number;
    tenantID?: number;
    salaryMonth?: number;
    salaryYear?: number;
    locked: boolean;
    lockDate?: Date;
    jvLocked: boolean;

}